src/endpoint/iam/iam_utils.js

```diff
@@ -79,67 +80,6 @@ function parse_max_items(input_max_items) {
     return value_as_number;
 }
 
-
-/**
- * _type_check_input checks that the input is the same as needed
- * @param {string} input_type
- * @param {string | number} input_value
- * @param {string} parameter_name
- */
-function _type_check_input(input_type, input_value, parameter_name) {
```

Comment:
i'm not quite sure that moving these functions to nc_utils.js make sense, because we are going to support IAM on containerized flow as well

Reply:
I think that iam_utils is the correct place for these functions, why can't the CLI use it from here?